fal_bot/consts.py

```diff
@@ -15,111 +15,282 @@
 ]
 
 FOOOCUS_STYLES = [
-    "sai-base",
```

Comment:
plz tell me these are all automatically generated :sob:

Reply:
in the land of the fooocus repository where diffusion models lie
one print to find them all